.travis.yml

```diff
@@ -12,10 +12,5 @@ addons:
 env:
   global:
   - NOKOGIRI_USE_SYSTEM_LIBRARIES=true # speeds up installation of html-proofer
-jobs:
-  include:
-    - stage: build
-      script: bundle exec jekyll build
-    - stage: testlinks
-      script: bundle exec jekyll build && bundle exec htmlproofer ./_site --check-html
-    
\ No newline at end of file
+before_script: bundle exec jekyll build
+script: bundle exec jekyll build && bundle exec htmlproofer --url-ignore "/cslabs" ./_site --check-html
```

Comment:
Does the build need to be run twice since it's being run before the script?

Reply:
Good point, I'll get that fixed